src/completions.ts

```diff
@@ -213,7 +215,10 @@ export abstract class CompletionSourceFuse extends CompletionSource {
 
         // Filter by query if query is not empty
         if (query) {
-            this.setStateFromScore(this.scoredOptions(query))
+            this.setStateFromScore(
+                this.scoredOptions(query, prefix === "taball " ? true : false),
+            )
```

Comment:
Surely this should be in `TabAll.ts`?

Need to look to see what it is actually doing. Why does `taball` need a special case that `tab` doesn't?